src/routes/projects/[id([0-9]+)]/index.svelte

```diff
@@ -43,6 +42,12 @@
     account != null
     && (account.is_admin || moderatorEmails.includes(account.email))
   );
+  $: firstActivities = (
+    $project.activities.slice(0, 3)
+      .filter(activity => !activity.internal)
```

Comment:
How about `filter` before `slice`? If the first few ones happen to be internal (we have nothing in the DB that prevents this, afaik), This will limit the visible activities

Reply:
True